config/locales/models/students/en.yaml

```diff
@@ -0,0 +1,5 @@
+en:
+  students:
+    errors:
+      group_creation_failure: "Sorry! For some reason, your group could not be created. Please wait a few seconds, then hit refresh to try again. If you come back to this page, you should inform the course instructor."
```

Comment:
From the other localizations files I saw, the text strings were generally one sentence. Also, since the `Please wait a few seconds...` part is repeated, would it be better if I extract that into a seperate text? I wasn't sure since they each represent different exceptions